suites/google/googleServiceAccountKeyTest.js

```diff
@@ -230,7 +230,10 @@ Scenario('SA key removal job test: remove expired creds that do not exist in goo
   let getCredsRes = await fence.do.getUserGoogleCreds(users.user0.accessTokenHeader);
   let credsList = getCredsRes.access_keys;
   let key = credsList.filter(key => key.name.includes(credsKey1))[0];
-  await google.deleteServiceAccountKey(key.name);
+  const deletionResult = await google.deleteServiceAccountKey(key.name);
+    chai.expect(
+	deletionResult instanceof Error, 'Google service account deletion must not return an Error'
```

Comment:
debugging would be easier if you included the error's description in the error message returned by `expect`... if it's possible to grab it from `deleteServiceAccountKey()`

Reply:
Brilliant suggestion @paulineribeyre ! I will push another change.

Reply:
i'm not sure at returns the errors as it is https://github.com/uc-cdis/gen3-qa/blob/aa222a5c2c8c26a82b886a601c3e6b40905695e3/utils/google.js#L398-L400

Reply:
NICE TEAM WORK! 
#oneTeam!